File: C# Unity Build/Dragon_Raid/Assets/Scripts/EnemyData.cs
using UnityEngine;

[CreateAssetMenu(fileName = "New Enemy Data", menuName = "Game/Enemy Data")]
public class EnemyData : ScriptableObject
{
    public string enemyName;
    public int maxHealth;
    public int attack;  // (ATK)
    public int defense; // (DEF)
    public int luck;    // (LCK)
    public int power;   // (PWR) 
}

File: DragonRR/DragonRR/src/commonMain/kotlin/samples/mygame/enemyData/enemyData.kt
package com.yourgame.data
import kotlinx.serialization.Serializable
@Serializable
data class EnemyData(
    val enemyName: String = "",
    val maxHealth: Int = 0,
    val attack: Int = 0,  // (ATK)
    val defense: Int = 0, // (DEF)
    val luck: Int = 0,    // (LCK)
    val power: Int = 0    // (PWR)
)
